client/src/utils.js

```diff
@@ -0,0 +1,17 @@
+// src: https://davidwalsh.name/cancel-fetch
+// docs: https://developer.mozilla.org/en-US/docs/Web/API/AbortController
+const abortableFetch = (request, opts) => {
+    const controller = new AbortController();
+    const signal = controller.signal;
+
+    return {
+        abort: () => controller.abort(),
+        ready: fetch(request, { ...opts, signal }),
+    };
+};
+
+// If we know it's been canceled! do nothing
+const consoleErrorNonAbortErrors = (e) =>
+    e.name !== "AbortError" && console.error(e);
```

Comment:
While it's fine for testing, I don't think we should plan on console logging errors. Depending on how we want to handle it, I think we should either show an error at the component level, or just ignore errors of this nature

Reply:
Should I change it to `const consoleErrorNonAbortErrors = () => {}`

Reply:
I think we can just remove that function altogether. It's just being used inside the try-catch, so we just leave the catch block empty except for a comment explaining that we don't want to notify the user of any errors there

Reply:
Fixed it!